config/config.php

```diff
@@ -17,5 +17,5 @@
     $services->load('Rector\\Laravel\\', __DIR__ . '/../src')
         ->exclude([__DIR__ . '/../src/{Rector,ValueObject}']);
 
-    $services->set(RenameClassNonPhpRector::class);
+    $rectorConfig->rule(RenameClassNonPhpRector::class);
```

Comment:
@TomasVotruba this needs manual change as well as $services uses at set load 

```

    $services->defaults()
        ->public()
        ->autowire()
        ->autoconfigure();
    $services->load('Rector\\Laravel\\', __DIR__ . '/../src')
        ->exclude([__DIR__ . '/../src/{Rector,ValueObject}']);
```

Reply:
On rector-symfony usage, it cause error:

```
  rector-symfony git:(main) ✗ vendor/bin/rector process config --dry-run

                                                                                                                        
 [ERROR] Call to undefined method "Rector\Config\RectorConfig::set()" in                                                
         /Users/samsonasik/www/rector-symfony/config/config.php (which is being imported from                           
         "/Users/samsonasik/www/rector-symfony/rector.php").
```

it seems needs to be reverted to use `$services->set()` to avoid same error.

Reply:
:+1: